src/components/media-player/ha-media-player-browse.ts

```diff
@@ -678,11 +696,16 @@ export class HaMediaPlayerBrowse extends LitElement {
         }
 
         .child .play {
+          transition: color 0.5s;
+          border-radius: 50%;
+          bottom: calc(50% - 24px);
+          right: calc(50% - 24px);
```

Comment:
Oh, this was an experiment, not happy with it yet. Would like the old behavior when it is not expandable, show the play button on hover.